frontend/js/src/explore/art-creator/ArtCreator.tsx

```diff
@@ -416,6 +420,24 @@ export default function ArtCreator() {
               Check out the #ListenbrainzMonday tag on your social platform of
               choice!
             </p>
+            <FontAwesomeIcon
```

Comment:
I'm not convinced of the placement here when we have a big toolbar on the page specifically for this sort of thing:
![image](https://github.com/user-attachments/assets/e0684b77-3dbb-49ba-a124-e20392481f2d)

I imagine it was done this way to make it the same as some other pages with the sidebar (i.e the fresh releases page)?